### PR TITLE
Couchbase tests with testcontainers

### PR DESCRIPTION
**Pull request by sandrogiacom**
Changed Couchbase tests to use testcontainers

### CODE REVIEW

**Comment by genie-jnosql** [comment]
Can one of the admins verify this patch?

File: couchbase-driver/src/test/java/org/jnosql/diana/couchbase/configuration/CouchbaseContainer.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ */
+package org.jnosql.diana.couchbase.configuration;
+
+import org.testcontainers.containers.FixedHostPortGenericContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+
+public class CouchbaseContainer {
```
Comment (bsideup) [inline couchbase-driver/src/test/java/org/jnosql/diana/couchbase/configuration/CouchbaseContainer.java]: FYI there is `CouchbaseContainer` in Testcontainers

Comment (sandrogiacom) [reply couchbase-driver/src/test/java/org/jnosql/diana/couchbase/configuration/CouchbaseContainer.java]: Oh sorry. I'll check that out. Thanks!

Comment (sandrogiacom) [reply couchbase-driver/src/test/java/org/jnosql/diana/couchbase/configuration/CouchbaseContainer.java]: I tried using testcontainers, but it was not possible due to a problem with Fts. Jnosql uses the FullTextIndex service. I think about closing this PR and continuing the discussion on testcontainers project. What do you think?

**Comment by otaviojava** [comment]
Hey @sandrogiacom thank you for the help.
I can merge it and then create a new PR with refactoring.

**Pull request merged**